### PR TITLE
Assist users reading the KNX integration page with KNX onboarding

### PR DESCRIPTION
**Pull request by b-uwe**
## Proposed change

Assist users reading the KNX integration page with KNX onboarding

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/knx.markdown [hunk 1]
```diff
@@ -70,6 +70,10 @@ There is currently support for the following device types within Home Assistant:
 - [Time](#time)
 - [Weather](#weather)
 
+## Free KNX online training
+
+For Home Assistant KNX users, you can start a FREE KNX online training & get a discounted ETS Home license [here](https://www.knx.org/knx-en/for-your-home/home-assistant/)
```
Comment (c0ffeeca7) [inline source/_integrations/knx.markdown]:
```suggestion
As a Home Assistant KNX user, you can start a FREE KNX online training and get a discounted ETS Home license on the [KNX website](https://www.knx.org/knx-en/for-your-home/home-assistant/).
```

**Review by c0ffeeca7 — APPROVED** [review]
LGTM, @b-uwe :+1:

**Review by farmio — APPROVED** [review]
Nice!

**Comment by b-uwe** [comment]
Thanks for merging @c0ffeeca7 😁 🙏

**Pull request merged**